src/site/_filters/language-list.js

```diff
@@ -15,52 +15,55 @@
  */
 
 const {
-  defaultLanguage,
   languageNames,
+  defaultLanguage,
   languageOrdering,
-} = require('../../../lib/utils/language');
+} = require('../../lib/utils/language');
+
 const {
   getDefaultUrl,
-  getLocaleSpecificUrl,
   getTranslatedUrls,
-} = require('../../_filters/urls');
-const {i18n} = require('../../_filters/i18n');
+  getLocaleSpecificUrl,
+} = require('../_filters/urls');
+
+/**
+ * Look up all supported languages for a specific post
+ *
+ * @param {String} lang The current lang of a post to look up.
+ * @param {EleventyCollectionItem} post The post to look up.
+ * @return {String[][]} The list of supported languages for a specific post
+ */
 
-module.exports = (url, lang) => {
-  const langHrefs = getTranslatedUrls(url).filter(
+const supportedlanguages = (post, lang) => {
+  const url = post.url;
+  const languages = getTranslatedUrls(url).filter(
     (langHref) => langHref[0] !== lang,
   );
 
   // Exit early if there are no translations.
-  if (langHrefs.length === 0) {
-    return '';
-  }
+  if (languages.length === 0) return;
```

Comment:
That essentially is `return undefined` which is not a valid return type according to the JSDoc. This should probably be `return []`.